apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -374,8 +375,15 @@ export const submitChatContents = createAsyncThunk(
   async (newUserMessageText: string, thunkAPI) => {
     const dispatch = thunkAPI.dispatch as AppDispatch;
     const state = thunkAPI.getState() as RootState;
-    const {savedAiCustomizations: aiCustomizations, chatEventsCurrent} =
-      state.aichat;
+    const {chatEventsCurrent, savedAiCustomizations} = state.aichat;
+    const levelSystemPrompt = (
+      state.lab.levelProperties as AichatLevelProperties | undefined
+    )?.aichatSettings?.levelSystemPrompt;
+    const aiCustomizations = {
+      ...savedAiCustomizations,
+      systemPrompt:
+        levelSystemPrompt + ' ' + savedAiCustomizations.systemPrompt,
```

Comment:
Prepend level system prompt to the student systemp prompt before sending request to aichat `chat_completion`.

Note that we current check for profanity via WebPurify when the student AI customizations are saved to S3.